package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "devDependencies": {
+        "@symfony/webpack-encore": "^0.29.0",
+        "core-js": "^3.0.0",
+        "regenerator-runtime": "^0.13.2",
+        "webpack-notifier": "^1.6.0"
+    },
+    "license": "UNLICENSED",
```

Comment:
Maybe set this to the Apache 2.0 license?

Reply:
Sure, no problem.